### PR TITLE
[MED-22][external] import nifti to raster layer mask annotations

### PR DESCRIPTION
**Pull request by tomvars**
# Problem
We previously supported nifti imports as polygons, this PR introduces support for nifti imports into the raster layer.
# Solution
Users now have a new mode available at upload. In addition to `video` and `instances` which both write polygons into Darwin, users now have `mask` mode which writes to the raster layer and creates `mask` annotations.

# Changelog
We've introduced the ability to import nifti labels directly into the raster layer! This means users can perform inference off-platform or import existing annotations into a project and then annotate them using our Masks feature.

### CODE REVIEW

File: darwin/importer/formats/nifti.py [hunk 1]
```diff
@@ -177,8 +196,96 @@ def get_video_annotation(
         raise Exception("If is_mpr is True, slot_names must be of length 3")
 
 
-def nifti_to_video_annotation(
-    volume, class_name, class_idxs, slot_names, view_idx=2, pixdims=(1, 1, 1)
+def get_mask_video_annotations(volume, processed_class_map, slot_names):
```
Comment (Nathanjp91) [inline darwin/importer/formats/nifti.py:199]: Can you type annotate this? Inputs and outputs.

File: darwin/importer/formats/nifti.py [hunk 2]
```diff
@@ -177,8 +196,96 @@ def get_video_annotation(
...
+            interpolated=False,
+            slot_names=slot_names,
+        )
+        mask_video_annotation.id = mask_annotation_ids[class_name]
+        mask_video_annotations.append(mask_video_annotation)
+
+    return [raster_video_annotation] + mask_video_annotations
+
+
+def nifti_to_video_polygon_annotation(
```
Comment (Nathanjp91) [inline darwin/importer/formats/nifti.py:282]: Same as above.

File: darwin/importer/formats/nifti.py [hunk 3]
```diff
@@ -376,3 +483,16 @@ def process_nifti(
         data_array = nib.orientations.apply_orientation(img.get_fdata(), ornt)
         pixdims = img.header.get_zooms()
         return data_array, pixdims
+
+
+def convert_to_dense_rle(raster):
```
Comment (Nathanjp91) [inline darwin/importer/formats/nifti.py:488]: as above.

**Review by Nathanjp91 — CHANGES_REQUESTED** [review]
Would also love to see some tests added if possible.

**Comment by tomvars** [comment]
> Would also love to see some tests added if possible.

Added a test

File: tests/darwin/importer/formats/import_nifti_test.py [hunk 4]
```diff
@@ -1,3 +1,4 @@
+import ast
```
Comment (Nathanjp91) [inline tests/darwin/importer/formats/import_nifti_test.py:1]: I'm guessing this a stray import from vscode? if so remove it
```suggestion
```

**Review by Nathanjp91 — APPROVED** [review]

**Pull request merged**